docs/reference/buildx_create.md

```diff
@@ -123,6 +123,8 @@ Passes additional driver-specific options. Details for each driver:
       ```
 - `kubernetes`
     - `image=IMAGE` - Sets the container image to be used for running buildkit.
+    - `withqemu=(true|false)` - Install QEMU emulation for multi platforms support.
+    - `qemuimage=IMAGE` - Sets the QEMU emulation image. Defaults to `tonistiigi/binfmt:latest`
```

Comment:
nit: what about renaming them to `qemu.install` and `qemu.image`

Reply:
Sure. looks better.